src/components/header/header.js

```diff
@@ -1,11 +1,31 @@
 import React from 'react'
 import style from './header.scss'
+import IconMenu from 'material-ui/IconMenu';
+import MenuItem from 'material-ui/MenuItem';
+import IconButton from 'material-ui/IconButton';
+import MoreVertIcon from 'material-ui/svg-icons/navigation/more-vert';
+import MuiThemeProvider from 'material-ui/styles/MuiThemeProvider';
+import getMuiTheme from 'material-ui/styles/getMuiTheme';
+import darkBaseTheme from 'material-ui/styles/baseThemes/lightBaseTheme';
 
 export default class Header extends React.Component{
   render(){
     return(
       <div id="header">
-        <h1>{this.props.heading}</h1>
+        <MuiThemeProvider muiTheme={getMuiTheme(darkBaseTheme)}>
```

Comment:
MuiThemeProvider- i think will be on top of your app, no need to repeat every where